bpf/lib/common.h

```diff
@@ -304,8 +304,8 @@ struct tunnel_key {
 		union v6addr	ip6;
 	};
 	__u8 family;
-	__u8 cluster_id;
-	__u16 pad;
+	__u8 pad;
```

Comment:
For the C struct this would be fine, but the mirrored Go struct isn't packed and throws off the alignment:
```
--- FAIL: TestCheckStructAlignments (0.00s)
    alignchecker_test.go:16: type TunnelKey's binary.Size (20) does not equal its unsafe.Sizeof (22) size (struct with implicit trailing padding?)
FAIL
```